craft_providers/lxd/remotes.py

```diff
@@ -152,10 +153,16 @@ def add_remote(self, lxc: LXC) -> None:
         remote_address=DAILY_REMOTE_ADDRESS,
         remote_protocol=ProtocolType.SIMPLESTREAMS,
     ),
+    centos.CentOSBaseAlias.SEVEN.value: RemoteImage(
```

Comment:
Feel free to refactor the `is_stable` property in this file if you want centos 7 to be considered stable.

Admittedly, I'm not happy with how I decided to implement remote names and remote urls in this file, but we're stuck with it for the short term.

Reply:
It probably not that stable. Those image from LXD are daily images.